Loaded dependency [python3/3.7.11]: gcc/10.3.0-binutils-2.36.1
Loaded module: python3/3.7.11

Loading python3/3.7.11
  Loading requirement: gcc/10.3.0-binutils-2.36.1
Loaded module: cuda/11.3
Loaded module: cudnn/v8.2.0.53-prod-cuda-11.3
Loaded module: ffmpeg/4.2.2
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /zhome/fa/4/127501/.local/include/python3.7m/UNKNOWN
sysconfig: /zhome/fa/4/127501/.local/include/python3.7
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: You are using pip version 21.1; however, version 21.3.1 is available.
You should consider upgrading via the '/appl/python/3.7.11/bin/python3 -m pip install --upgrade pip' command.
  WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
  distutils: /zhome/fa/4/127501/.local/include/python3.7m/importlib-metadata
  sysconfig: /zhome/fa/4/127501/.local/include/python3.7/importlib-metadata
  WARNING: Additional context:
  user = True
  home = None
  root = None
  prefix = None
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /zhome/fa/4/127501/.local/include/python3.7m/UNKNOWN
sysconfig: /zhome/fa/4/127501/.local/include/python3.7
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: You are using pip version 21.1; however, version 21.3.1 is available.
You should consider upgrading via the '/appl/python/3.7.11/bin/python3 -m pip install --upgrade pip' command.
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /zhome/fa/4/127501/.local/include/python3.7m/UNKNOWN
sysconfig: /zhome/fa/4/127501/.local/include/python3.7
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: You are using pip version 21.1; however, version 21.3.1 is available.
You should consider upgrading via the '/appl/python/3.7.11/bin/python3 -m pip install --upgrade pip' command.
Loaded module: cuda/11.3
Loaded module: cudnn/v8.2.0.53-prod-cuda-11.3
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /zhome/fa/4/127501/.local/include/python3.7m/UNKNOWN
sysconfig: /zhome/fa/4/127501/.local/include/python3.7
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: You are using pip version 21.1; however, version 21.3.1 is available.
You should consider upgrading via the '/appl/python/3.7.11/bin/python3 -m pip install --upgrade pip' command.
  WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
  distutils: /zhome/fa/4/127501/.local/include/python3.7m/importlib-metadata
  sysconfig: /zhome/fa/4/127501/.local/include/python3.7/importlib-metadata
  WARNING: Additional context:
  user = True
  home = None
  root = None
  prefix = None
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /zhome/fa/4/127501/.local/include/python3.7m/UNKNOWN
sysconfig: /zhome/fa/4/127501/.local/include/python3.7
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: You are using pip version 21.1; however, version 21.3.1 is available.
You should consider upgrading via the '/appl/python/3.7.11/bin/python3 -m pip install --upgrade pip' command.
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /zhome/fa/4/127501/.local/include/python3.7m/UNKNOWN
sysconfig: /zhome/fa/4/127501/.local/include/python3.7
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: You are using pip version 21.1; however, version 21.3.1 is available.
You should consider upgrading via the '/appl/python/3.7.11/bin/python3 -m pip install --upgrade pip' command.
Traceback (most recent call last):
  File "PPO.py", line 302, in <module>
    create_and_train_network()
  File "PPO.py", line 161, in create_and_train_network
    next_obs, reward, done, info = env.step(action)
  File "/zhome/fa/4/127501/DeepLearning/utils.py", line 259, in step
    return self.step_wait()
  File "/zhome/fa/4/127501/DeepLearning/utils.py", line 534, in step_wait
    obs, reward, done, info = self.venv.step_wait()
  File "/zhome/fa/4/127501/DeepLearning/utils.py", line 516, in step_wait
    obs, reward, done, info = self.venv.step_wait()
  File "/zhome/fa/4/127501/DeepLearning/utils.py", line 501, in step_wait
    obs, reward, done, info = self.venv.step_wait()
  File "/zhome/fa/4/127501/DeepLearning/utils.py", line 468, in step_wait
    obs, rews, news, infos = self.venv.step_wait()
  File "/zhome/fa/4/127501/DeepLearning/utils.py", line 340, in step_wait
    obs, rews, dones, infos = self.venv.step_wait()
  File "/zhome/fa/4/127501/.local/lib/python3.7/site-packages/gym3/interop.py", line 323, in step_wait
    rew, ob, first = self.env.observe()
  File "/zhome/fa/4/127501/.local/lib/python3.7/site-packages/gym3/libenv.py", line 345, in observe
    self._c_lib.libenv_observe(self._c_env)
KeyboardInterrupt
